contracts/strategies/88mph/base/EightyEightMphStrategy.sol

```diff
@@ -0,0 +1,676 @@
+pragma solidity 0.5.16;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/math/Math.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721Holder.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+import "../../../base/upgradability/BaseUpgradeableStrategyUL.sol";
+
+import "../../../base/interface/IStrategy.sol";
+import "../../../base/interface/IVault.sol";
+import "../../../base/PotPool.sol";
+import "../interface/IDInterest.sol";
+import "../interface/IDInterestLens.sol";
+import "../interface/IxMph.sol";
+import "../interface/IVesting.sol";
+
+contract EightyEightMphStrategy is IStrategy, BaseUpgradeableStrategyUL, ERC721Holder {
```

Comment:
Note that this strategy can hold NFTs because 88mph deposits and vesting are represented as NFTs. 
We don't have anything to gain from transferring them though so I just added the ERC721Holder